{
	"schema_version": "1.6.0",
	"id": "GHSA-mhhc-r88h-2qrm",
	"modified": "2023-11-08T04:00:02.359546Z",
	"published": "2022-05-14T01:03:18Z",
	"aliases": [
		"CVE-2018-16887"
	],
	"summary": "katello Cross-site Scripting vulnerability",
	"details": "A cross-site scripting (XSS) flaw was found in the katello component of Satellite. An attacker with privilege to create/edit organizations and locations is able to execute a XSS attacks against other users through the Subscriptions or the Red Hat Repositories wizards. This can possibly lead to malicious code execution and extraction of the anti-CSRF token of higher privileged users. Versions before 3.9.0 are vulnerable.",
	"severity": [
		{
			"type": "CVSS_V3",
			"score": "CVSS:3.0/AV:N/AC:L/PR:L/UI:R/S:C/C:L/I:L/A:N"
		}
	],
	"affected": [
		{
			"package": {
				"ecosystem": "RubyGems",
				"name": "katello",
				"purl": "pkg:gem/katello"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "3.9.0"
						}
					]
				}
			],
			"versions": [
				"1.5.0",
				"2.2.2",
				"2.4.0",
				"2.4.0.rc1",
				"2.4.0.rc2",
				"2.4.0.rc3",
				"2.4.1",
				"2.4.2",
				"2.4.3",
				"2.4.4",
				"2.4.5",
				"3.0.0",
				"3.0.0.rc1",
				"3.0.0.rc2",
				"3.0.0.rc3",
				"3.0.0.rc4",
				"3.0.0.rc5",
				"3.0.0.rc7",
				"3.0.1",
				"3.0.2",
				"3.1.0",
				"3.1.0.1",
				"3.1.0.rc1",
				"3.1.0.rc2.1",
				"3.2.0",
				"3.2.0.rc1",
				"3.2.0.rc1.1",
				"3.2.0.rc2",
				"3.2.0.rc3",
				"3.2.1",
				"3.2.1.1",
				"3.3.0",
				"3.3.0.1",
				"3.3.0.rc1",
				"3.3.0.rc1.1",
				"3.3.0.rc2",
				"3.3.1",
				"3.3.1.1",
				"3.3.2",
				"3.4.0",
				"3.4.0.1",
				"3.4.0.2",
				"3.4.0.rc1",
				"3.4.0.rc2",
				"3.4.1",
				"3.4.2",
				"3.4.4",
				"3.4.5",
				"3.5.0",
				"3.5.0.1",
				"3.5.0.rc1",
				"3.5.0.rc2",
				"3.5.1",
				"3.5.1.1",
				"3.5.2",
				"3.6.0",
				"3.6.0.1.rc2",
				"3.6.0.rc1",
				"3.6.0.rc2",
				"3.7.0",
				"3.7.0.rc1",
				"3.7.0.rc2",
				"3.7.1",
				"3.7.1.1",
				"3.8.0",
				"3.8.0.rc1",
				"3.8.0.rc2",
				"3.8.0.rc3",
				"3.8.1",
				"3.9.0.rc1",
				"3.9.0.rc2"
			],
			"database_specific": {
				"source": "https://github.com/github/advisory-database/blob/main/advisories/github-reviewed/2022/05/GHSA-mhhc-r88h-2qrm/GHSA-mhhc-r88h-2qrm.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2018-16887"
		},
		{
			"type": "WEB",
			"url": "https://access.redhat.com/errata/RHSA-2019:1222"
		},
		{
			"type": "WEB",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-16887"
		},
		{
			"type": "PACKAGE",
			"url": "https://github.com/Katello/katello"
		},
		{
			"type": "WEB",
			"url": "https://github.com/rubysec/ruby-advisory-db/blob/master/gems/katello/CVE-2018-16887.yml"
		}
	],
	"database_specific": {
		"cwe_ids": [
			"CWE-79"
		],
		"github_reviewed": true,
		"github_reviewed_at": "2023-01-27T00:11:54Z",
		"nvd_published_at": "2019-01-13T02:29:00Z",
		"severity": "MODERATE"
	}
}
